src/Security/Authentication/WsFederation/src/WsFederationOptions.cs

```diff
@@ -181,4 +205,13 @@ public TokenValidationParameters TokenValidationParameters
     /// </summary>
     [EditorBrowsable(EditorBrowsableState.Never)]
     public new bool SaveTokens { get; set; }
+
+    /// <summary>
+    /// Gets of sets the <see cref="UseTokenHandlers"/> property to control if <see cref="TokenHandlers"/> or <see cref="SecurityTokenHandlers"/> will be used to validate the inbound token.
+    /// The advantage of using the TokenHandlers is:
+    /// <para>There is an Async model.</para>
+    /// <para>The default token handler is a <see cref="JsonWebTokenHandler"/> which is 30 % faster when validating.</para>
```

Comment:
"The default token handler is" is kind of incorrect, since there are 3 default TokenHandlers. Maybe just a slight re-wording would help:

```suggestion
    /// <para>The default JWT token handler is a <see cref="JsonWebTokenHandler"/> which is 30 % faster when validating.</para>
```

Reply:
Addressed: 507d24137b37c43ddf9c18bdeb7510e70c84bcf1